### PR TITLE
Add mozorg/home.ftl

### PR DESCRIPTION
**Pull request by alexgibson**
Ref: https://github.com/mozilla/bedrock/pull/9919

### CODE REVIEW

File: en/mozorg/home.ftl [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/ ('en-US', 'de' and 'fr' excluded)
+
+home-internet-for-people-not-profit = Internet for people, not profit
+home-did-you-know-mozilla-the-maker = Did you know? { -brand-name-mozilla } — the maker of { -brand-name-firefox } — fights to keep the Internet a global public resource, open and accessible to all.
```
Comment (peiying2) [inline en/mozorg/home.ftl:8]: `Internet` - lower case

File: en-GB/mozorg/home.ftl [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+### URL: https://www-dev.allizom.org/ ('en-US', 'de' and 'fr' excluded)
```
Comment (peiying2) [inline en-GB/mozorg/home.ftl:6]: seems like en-CA, en-GB are excluded too, by looking at the staging server.

**Review by peiying2 — CHANGES_REQUESTED** [review]
Looks good. Minor fix request.

**Comment by alexgibson** [comment]
@peiying2 thanks updated

File: en/mozorg/home.ftl [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+home-fast-for-good = Fast for good.
+home-with-2x-the-speed-built-in = With 2x the speed, built-in privacy protection and { -brand-name-mozilla } behind it, the new { -brand-name-firefox } is the better way to browse.
+home-we-make-the-internet-safer = We make the internet safer, healthier and faster for good.
+home-mozilla-is-the-not-for-profit = { -brand-name-mozilla } is the not-for-profit behind { -brand-name-firefox }, the original alternative browser. We create products and policy to keep the internet in service of people, not profit.
+home-our-impact = Our impact
+home-when-you-use-firefox-you-help = When you use { -brand-name-firefox }, you help { -brand-name-mozilla } fight misinformation online, teach digital skills and make the comments section more human. Check out what helps create a healthier internet.
+home-working-to-keep-the-internet = Working to keep the internet healthy, open and accessible to all, we teach web literacy, provide tools and advocate on behalf of every individual who values the Internet as a global public resource.
```
Comment (peiying2) [inline en/mozorg/home.ftl:15]: Ooops, one more internet to fix at the end of this sentence: ...`who values the Internet as a global public resource.`

**Review by peiying2 — CHANGES_REQUESTED** [review]
Sorry, I missed one more capital initial for "internet'. This should be it, :).

**Comment by alexgibson** [comment]
@peiying2 good catch updated

**Review by peiying2 — APPROVED** [review]
Thank you! Good to go!

**Pull request merged**